agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -52,13 +56,42 @@ while true; do
 			shift;
 		fi
 		;;
+		--sysinfo)
+		shift;
+		if [ -n "$1" ]; then
+			sysinfo="$1"
+			shift;
+		fi
+		;;
+		--check)
+		shift;
+		if [ -n "$1" ]; then
+			check="$1"
+			shift;
+		fi
+		;;
 		--)
 		shift;
 		break;
 		;;
 	esac
 done
 
+if [ "$sysinfo" == "none" ]; then
+	exit 0
+fi
+
+if [ "$check" == "true" ]; then
+        for item in ${sysinfo//,/ }; do
```

Comment:
Won't this fail for `all` and `default`?

I think we need to check for `all` and `default`, and convert them to a list.  For now, `all` and `defaults` should equate to the list in `sysinfo_opts`.